src/main/webapp/app/complaints/list-of-complaints/list-of-complaints.component.ts

```diff
@@ -98,6 +102,9 @@ export class ListOfComplaintsComponent implements OnInit {
             }
         }
         this.subscribeToComplaintResponse(complaintResponse);
+        this.courseManagementService.find(this.courseId).subscribe((response) => {
+            this.course = response?.body!;
+        });
```

Comment:
Should we maybe do this inside the subscribe where we receive the courseId?

Reply:
Yes, it gets used above this call to create an Observable for the complaints. This allows us to run both requests in parallel (we already have the ID so why wait for the whole course object to then read the id?).

Reply:
The code above also assumes that the id is present so we should be fine, but I'll have another look.